git/README.md

```diff
@@ -0,0 +1,44 @@
+# Git
+
+Git is pretty awesome. Here's some reasonable standards when it comes to using it.
+
+## Commit Messages
+
+At Mobify, we recommend a few standards regarding git commit messages:
+- If the commit is related to a JIRA ticket, prefix the case-sensitive ticket number to the start of the message
```

Comment:
I don't think requiring a JIRA ticket number in each commit message fits inside the workflow of many of the product teams particularly. I do understand why traceability is a huge asset for figuring out why change X was made, especially from a maintenance perspective.

I think it's pretty reasonable to ask for branches + PRs to have a JIRA ticket number in them. But even that, sometimes I create branches that don't have a JIRA ticket. This can't be a rule, but maybe a guideline. An example, if I'm going to fix 2-3 typos in some docs on astro.mobify.com, I'm not going to create a JIRA ticket for that. As a general rule on our team we aren't creating JIRA tickets for things that come up that take less than ~30 minutes since creating a ticket actually adds 2+ minutes to this which is a non-trivial percentage of the actual work involved.

IMO -- we shouldn't need to change our process to fit the tool. If Github + JIRA integration requires changing commit messages in a way that is going to slow down the entire team, we should modify the tool, not our process.

RE partners: We already have enough to train partners on and even with our better partners we've had a bit of an uphill battle (in my experience) to get them to open PRs at the correct time and use good branching strategy. If partners are building all of our projects then these rules really apply to them more than Mobify-internal. I'm concerned that they have enough to learn already with the Mobify platform that this might not be a battle worth fighting just yet.